Libraries/LibThread/Thread.h

```diff
@@ -41,12 +42,14 @@ class Thread final : public Core::Object {
     virtual ~Thread();
 
     void start();
-    void quit(void *code = 0);
+    void quit(void* code = 0);
```

Comment:
It's not cool to use 0 as a pointer, not even by C standards :slightly_smiling_face:

Reply:
Fair enough that is how it is defined but I can change it to nullptr it will resolve to the same thing so it shouldn't break compatibility.

Reply:
Again, compatibility with what? This is new code